kms/encrypt_symmetric.go

```diff
@@ -40,21 +41,41 @@ func encryptSymmetric(w io.Writer, name string, message string) error {
 	// Convert the message into bytes. Cryptographic plaintexts and
 	// ciphertexts are always byte arrays.
 	plaintext := []byte(message)
+	
+	// Optional, but recommended: Compute plaintext's CRC32C.
+	// See crc32c() function below.
+	plaintextCrc32c = crc32c(plaintext)
```

Comment:
```suggestion
	plaintextCRC32C = crc32c(plaintext)
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms